ui/app/mainui/SendModalHandler.qml

```diff
@@ -240,32 +325,79 @@ QtObject {
             preSelectedHoldingType: tokenType,
             onlyAssets: true
         }
-        openSend(params)
+        openSend(params, true)
     }
 
     function sendToken(senderAddress, tokenId, tokenType) {
         let sendType = Constants.SendType.Transfer
+        let selectedChainId = 0
+        let selectedRawAmount = ""
         if (tokenType === Constants.TokenType.ERC721)  {
             sendType = Constants.SendType.ERC721Transfer
-        } else if(tokenType === Constants.TokenType.ERC1155) {
+            selectedRawAmount = "1"
+            selectedChainId =
+                    SQUtils.ModelUtils.getByKey(root.collectiblesBySymbolModel, "symbol", tokenId, "chainId")
+        }
+        else if(tokenType === Constants.TokenType.ERC1155) {
             sendType = Constants.SendType.ERC1155Transfer
+            selectedRawAmount = "1"
+            selectedChainId =
+                    SQUtils.ModelUtils.getByKey(root.collectiblesBySymbolModel, "symbol", tokenId, "chainId")
         }
-        let params = {
-            preSelectedSendType: sendType,
-            preSelectedAccountAddress: senderAddress,
-            preSelectedHoldingID: tokenId ,
-            preSelectedHoldingType: tokenType,
+        else {
+            let layer1chainId = SQUtils.ModelUtils.getByKey(root.filteredFlatNetworksModel, "layer", "1", "chainId")
+            let networksChainIdArray = SQUtils.ModelUtils.modelToFlatArray(root.filteredFlatNetworksModel, "chainId")
```

Comment:
here we're defining `networksChainIdArray`, but below use it as `root.networksChainIdArray`.